src/components/MultiHopTrade/hooks/quoteValidation/useActiveQuoteStatus.tsx

```diff
@@ -96,10 +101,18 @@ export const useActiveQuoteStatus = (): QuoteStatus => {
           return 'trade.errors.quoteError'
         case ActiveQuoteStatus.NoQuotesAvailable:
           return 'trade.errors.noQuotesAvailable'
-        case ActiveQuoteStatus.SellAmountBelowMinimum:
-          return selectedSwapperName && [SwapperName.LIFI].includes(selectedSwapperName)
-            ? 'trade.errors.amountTooSmallOrInvalidTradePair'
-            : ['trade.errors.amountTooSmall', { minLimit: minimumAmountUserMessage }]
+        case ActiveQuoteStatus.SellAmountBelowMinimum: {
+          const details = (activeQuoteError?.details ?? {}) as { minAmountCryptoPrecision?: string }
```

Comment:
Prefer type annotation (explicitly defining types) over type casting (changing an object's type):


```suggestion
          const details: { minAmountCryptoPrecision?: string } = activeQuoteError?.details ?? {}
```

Reply:
agreed, i prefer that too. I'll update it